compair/tests/api/test_comparisons.py

```diff
@@ -552,6 +567,10 @@ def test_score_calculation(self, mock_shuffle):
         This is just a rough check on whether score calculations are correct. Answers
         that has more wins should have the highest scores.
         """
+        # For the logic "winning more rounds -> higher score" to work in this particular
+        # test case, we are adding one more answer (from the default of 2 to 3) to the assignment
+        self._revise_number_of_answer(num_answer=3, num_draft_answers=1)
```

Comment:
If all that's needed is another answer in the pool to make this work, maybe instead you could just add
```
self.data.create_answer(self.assignment, self.data.get_authorized_instructor())
```

Reply:
The additional answer needs to be in the comparable pool.  So when the test case completes all comparisons for primary and secondary authorized students, answers with more wins will have higher scores.

Reply:
An answer added this way should be comparable though (answers are comparable by default).

I think I see the problem. Can you remove `self.comparable_answers` for ComparisonTestData and change `get_comparable_answers` to
```
return [answer for answer in self.answers if answer.comparable]
```

Reply:
Changed that.  Also fixed the issue of duplicate answers being added to the list `self.answers`